.github/workflows/sync-hugo.yaml

```
@@ -0,0 +1,28 @@
+# This workflow runs the make sync-hugo-version command and alters the dependabot pr
+name: Hugo Version Synchronizer
+
+on:
+  pull_request_target:
+    branches: [ main ]
+
+jobs:
+    run-sync-hugo-version:
+        if: ${{ github.actor == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependabot') }}
```

```suggestion
        if: ${{ github.actor == 'dependabot[bot]' }}
```

Are you sure we have this label?